frame/democracy/src/tests.rs

```diff
@@ -49,6 +49,8 @@ const NAY: Vote = Vote { aye: false, conviction: Conviction::None };
 const BIG_AYE: Vote = Vote { aye: true, conviction: Conviction::Locked1x };
 const BIG_NAY: Vote = Vote { aye: false, conviction: Conviction::Locked1x };
 
+const MAX_PROPOSALS: u32 = 100;
```

Comment:
why redefining MAX_PROPOSAL here instead of using the crate MAX_PROPOSAL ?

Reply:
It is now used.